diff --git a/Smarty/templates/Header.tpl b/Smarty/templates/Header.tpl
index 581013c..8988ce4 100755
--- a/Smarty/templates/Header.tpl
+++ b/Smarty/templates/Header.tpl
@@ -96,7 +96,7 @@
- 	{if $MODULE_NAME neq 'FieldFormulas' && $MODULE_NAME neq 'com_vtiger_workflow' && $MODULE_NAME neq 'NgBlock' $MODULE_NAME neq 'ESClient' && $MODULE_NAME neq 'evvtApps' && $MODULE_NAME neq 'VtappSecurity' && $MODULE_NAME neq 'Settings' && $MODULE_NAME neq  'ElasticSearch' }   
+       {if $MODULE_NAME neq 'FieldFormulas' && $MODULE_NAME neq 'com_vtiger_workflow' && $MODULE_NAME neq 'NgBlock' && $MODULE_NAME neq 'FormBuilder' && 
        <script type="text/javascript" src="Smarty/angular/angular.js"></script>
        <script  src="Smarty/angular/ng-table.js"></script>
        <script src="Smarty/angular/angular-multi-select.js"></script>  
        <link rel="stylesheet" href="Smarty/angular/angular-multi-select.css">

diff --git a/Smarty/templates/Header.tpl b/Smarty/templates/Header.tpl
index 581013c..8988ce4 100755
--- a/Smarty/templates/Header.tpl
+++ b/Smarty/templates/Header.tpl
@@ -96,7 +96,7 @@

{* END *}
-    {if $MODULE_NAME neq 'FieldFormulas' && $MODULE_NAME neq 'com_vtiger_workflow' && $MODULE_NAME neq 'NgBlock' && $MODULE_NAME neq 'ESClient' && $MODULE_NAME neq 'evvtApps' && $MODULE_NAME neq 'VtappSecurity' && $MODULE_NAME neq 'Settings' && $MODULE_NAME neq 'ElasticSearch'}   
+    {if $MODULE_NAME neq 'FieldFormulas' && $MODULE_NAME neq 'com_vtiger_workflow' && $MODULE_NAME neq 'NgBlock' && $MODULE_NAME neq 'FormBuilder' && 
        <body leftmargin=0 topmargin=0 marginheight=0 marginwidth=0 class=small ng-app="demoApp" ng-cloak {if $MODULE_NAME eq 'PointofSale' || $MODULE_NAME eq 'Distributor'} onload="autocompleteAddressPOS();" {/if}> 
            <script>
                angular.module('demoApp', ['ngTable','ui.bootstrap','multi-select','xeditable',
                                            'ngTagsInput','textAngular','ui.sortable','ngRoute']);
            </script>
